### PR TITLE
add new window override

### PR DESCRIPTION
**Pull request by DeJayDev**
Closes #13 by adding `chrome_settings_overrides` to the manifest.json

Documentation with Version Support Reference:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/chrome_settings_overrides

## Summary by CodeRabbit

- **New Features**
	- Introduced a homepage override for the browser extension, directing users to a specified HTML file.

- **Documentation**
	- Updated the manifest to include new settings for improved user experience.

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
   "chrome_url_overrides": {
     "newtab": "index.html"
   },
+  "chrome_settings_overrides": {
+    "homepage": "index.html"
+  },
```
[the remark this answers is not in the document]
Comment (DeJayDev) [reply public/manifest.json:11]: Please ignore this comment, `browser_specific_settings` is specifically only ever used by Firefox (and Firefox Android). It is used to define a minimum and maximum Gecko version, an update URL for users not using AMO, and a custom ID which is required by Manifest v3 extensions.

**Comment by DeJayDev** [comment]
Like code rabbits nitpick comment, the label is wrong too <3 This setting is respected by Firefox and Chrome.

**Comment by DeJayDev** [comment]
I converted this to a draft, but I think I'll close it with this. Firefox is fine with this setting but Chrome doesn't like that it's not a full URL. Others have done this by splitting the extension to use a Firefox specific manifest and a Chrome specific manifest.

**Comment by georg-stone** [comment]
Hey @DeJayDev ! I noticed your PR late, and I think this would be a great addition. I do have plans to set up a `npm run build:chrome` and `npm run build:firefox`, which will include specific manifests. Let me know if you'd be willing to reopen this and build out this feature.

**Comment by georg-stone** [comment]
@DeJayDev  I have recently added this into the newest release. Thank you for informing be about this feature!

**Comment by SpyC0der77** [comment]
@georg-stone "informing be" 💀

**Pull request closed without merging**